### PR TITLE
Updates ARM template files - Cognitive Services

### PR DESCRIPTION
**Pull request by davidsmatlak**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

- Updates template used in ARM template [quickstart](https://docs.microsoft.com/azure/cognitive-services/resource-manager-template) article.
- Adds quickstart article's author to `docOwner` in _metadata.json_.
- Updates _README.md_ with link to the quickstart article.
- Fixes property order based on best practices guidance.
- Reformats JSON file indentations from 4 spaces to 2 spaces.

cc: @mumian @tfitzmac

### CODE REVIEW

**Comment by bmoore-msft** [comment]
@aahill - check this PR for updates that may be needed to documentation that references this sample.

File: 101-cognitive-services-universalkey/azuredeploy.json [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
+      "sku": {
+        "name": "[parameters('sku')]"
+      },
+      "kind": "CognitiveServices",
+      "properties": {
+        "statisticsEnabled": false
+      }
+    }
+  ]
 }
```
Comment (davidsmatlak) [inline 101-cognitive-services-universalkey/azuredeploy.json:42]: @bmoore-msft should a blank line be added to fix the `no newline at end of file` message?

Comment (davidsmatlak) [reply 101-cognitive-services-universalkey/azuredeploy.json]: fixed.

**Comment by bmoore-msft** [comment]
@aahill - check this PR for updates that may be needed to documentation that references this sample.

**Comment by aahill** [comment]
Thanks @bmoore-msft, the article ingests the code from this repo, so once this PR is merged, I'll get the documentation to refresh and pull in the new changes.

**Comment by bmoore-msft** [comment]
@aahill - check this PR for updates that may be needed to documentation that references this sample.

**Pull request merged**